mysite/mysite/urls.py

```diff
@@ -34,13 +35,19 @@
 
 admin.autodiscover()
 
+# JSON Web Token Authentication
+jwt_patterns = ([
```

Comment:
You don't have to break urlpatterns into parts. Just put this directly in urlpatterns.